kibit/src/kibit/rules/collections.clj

```diff
@@ -20,5 +20,7 @@
   ;; set
   [(into #{} ?coll) (set ?coll)]
 
-  [(take ?n (repeatedly ?coll)) (repeatedly ?n ?coll)])
+  [(take ?n (repeatedly ?coll)) (repeatedly ?n ?coll)]
+  [(doall (map ?fn ?coll)) (run! ?fn ?coll)]
```

Comment:
I'd be happier about `dorun` here instead of `doall`, as `(doall (map` returns a sequence, and run! returns nil